homework/App.js

```diff
@@ -44,24 +48,26 @@ class App {
    * repo and its contributors as HTML elements in the DOM.
    * @param {number} index The array index of the repository.
    */
-  async fetchContributorsAndRender(index) {
+  async fetchContributorsAndRender(url) {
     try {
```

Comment:
When I first saw this variable name I thought it's all the repositories, while it contains a one only.